project/build.sbt

```diff
@@ -1,13 +1,11 @@
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
 
-addSbtPlugin("com.lucidchart" % "sbt-cross" % "3.0")
+addSbtPlugin("com.lucidchart" % "sbt-cross" % "4.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.9")
```

Comment:
I don't think this version of the play plugin is compatible with sbt 1.0+. But upgrading to a more recent version of play is probably going to be more work :(

Reply:
We've upgraded to play 2.8